### PR TITLE
fix: username and password provided in config.yaml is not working

### PR DESCRIPTION
**Pull request by boddumanohar**
### Changes

https://github.com/0chain/blobber/issues/856

### Fixes
call `SetAdminCredentials()` before going further.

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/blobber/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- 0chain:
- gosdk:
- system_test: https://github.com/0chain/system_test/pull/463
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | 856  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix-creds  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | 856  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

File: code/go/0chain.net/core/common/admin.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ func SetAdminCredentials() {
 }
 
 func AuthenticateAdmin(handler ReqRespHandlerf) ReqRespHandlerf {
+	SetAdminCredentials()
```
Comment (peterlimg) [inline code/go/0chain.net/core/common/admin.go]: Let's call this SetAminiCredentials()  [initHandlers](https://github.com/0chain/blobber/blob/4b3a59abe95e1713de7e75a4dc38fd97a002a262/code/go/0chain.net/blobber/http.go#L86) function, the problem of calling it here is we will update the global variables on every API calls.

**Review by peterlimg — CHANGES_REQUESTED** [review]

File: code/go/0chain.net/core/common/admin.go (hunk 1 above)
Comment (boddumanohar) [reply code/go/0chain.net/core/common/admin.go]: updated.

**Review by peterlimg — APPROVED** [review]

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | TRUE  |
| system-tests  | master  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | 856  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request merged**